scripts/golinter.sh

```diff
@@ -41,3 +53,18 @@ do
         exit 1
     fi
 done
+
+for i in "${goModules[@]}"
+do
+    echo ">>>Checking $i with go vet"
+    pushd $i
+        OUTPUT="$(GO111MODULE=on go vet .)"
+        exitCode=$?
+    popd
```

Comment:
this moves in and out of each listed module directory to run the vet command.

I wonder if the original invocations are compatible in the same way, to again merge the loops.

vet could probably be reduced to a ./... but not the other commands, so in the effort of sharing the same lists, this is fine. 🤷‍♂ 🤷‍♀